Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/ClustererTask.h

```diff
@@ -63,6 +63,8 @@ class ClustererTask
   void setMaxROframe(int max) { maxROframe = max; }
   int getMaxROframe() const { return maxROframe; }
 
+  void setPatterns() { mClusterer.setPatterns(&mPatterns); }
```

Comment:
Is it not dangerous to let the user set the pattern with a public method?

Reply:
I would say, in this context, it is no more dangerous than passing these mPatterns as an argument of other functions.  But adding mPatterns to functions' arguments would break the interface, and a few other places would needed additional fixes.
This solution (true, not very elegant) saves some extra coding. I would keep it.

Reply:
Ok, I agree